03_Modules/Cdrs/src/module/handlers.ts

```diff
@@ -62,11 +63,7 @@ export class CdrsOcppHandlers extends AbstractModule {
     const timer = new Timer();
     this._logger.info('Initializing...');
 
-    if (!deasyncPromise(this._initHandler(this._requests, this._responses))) {
-      throw new Error(
-        'Could not initialize module due to failure in handler initialization.',
-      );
-    }
+    this.initHandlers();
```

Comment:
How about init handlers centrally in the `Server/src/index.ts` like [here](https://github.com/citrineos/citrineos-core/blob/9432540f35b5efe0369e2f02dc1fb1eb032b88c0/Server/src/index.ts#L461) rather than in each `handlers`. I remember this is what @thanaParis suggested in the daily. Then we also don't need `00_Base/src/util/Timer.ts` in this case. (If I misunderstand or missed anything, pls feel free to let me know)